templates/contrib/jdbc/Makefile

```diff
@@ -33,9 +35,9 @@ sybase:
 
 postgresql:
 	sh prepare.sh postgresql
-	cekit --verbose build --overrides-file db-overrides.yaml ${BUILD_ENGINE} --tag=quay.io/kiegroup/jboss-kie-postgresql-extension-openshift-image:42.2.5
+	cekit --verbose build --overrides-file db-overrides.yaml ${BUILD_ENGINE}
 	cekit --verbose test --overrides-file db-overrides.yaml behave
-
+	${BUILD_ENGINE} tag kiegroup/jboss-kie-postgresql-extension-openshift-image:42.2.5st quay.io/kiegroup/jboss-kie-postgresql-extension-openshift-image:42.2.5
```

Comment:
I think there is a typo here in 42.2.5st

Reply:
yes, updated :)